.travis_install.bash

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+if [ "$TRAVIS_OS_NAME" == "linux" ]
+then
+    if [ "$CXX" = "g++" ];
+    then
+        sudo apt-get install -qq g++-9
+        export CXX="g++-9"
+    elif [ "$CXX" == "clang++" ]
+    then
+        sudo apt-get install -y  --allow-unauthenticated -qq clang-8 libomp-8-dev libc++-8-dev libc++abi-8-dev
```

Comment:
```suggestion
        sudo apt-get install -y --allow-unauthenticated -qq clang-8 libomp-8-dev g++-9
```
Since you are by default not using libc++ but rather libstdc+ you don't need those, instead add gcc-9 for a libstdc++ that supports filesystem.

Also: Consider removing `--allow-unauthenticated`, gpg-keys have been installed this really should work w.o. this flag

Reply:
makes sense. Thanks!